src/utils/icons.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import LinkIco from '@material-ui/icons/Launch';
+import HomeIco from '@material-ui/icons/Home';
+import TimerIco from '@material-ui/icons/Timer';
+import BatteryIco from '@material-ui/icons/BatteryChargingFull';
+import VideoIco from '@material-ui/icons/Videocam';
+import DesktopIco from '@material-ui/icons/DesktopWindows';
+import AndroidIco from '@material-ui/icons/PhoneAndroid';
+import DetailsIco from '@material-ui/icons/MoreHoriz';
+import CodeIco from '@material-ui/icons/Code';
```

Comment:
It's odd to leave the 'n' off of Icon here. I think it's better to have a longer name that's clearer, rather than a short vague one :) So `materialLinkIcon` or or rename your icon components to something like `LinkIconStyle`.

However, since some of the styling is the same, couldn't you create a function that returns the div with the style and pass in the Icon element or prop (for `LinkIcon` and `DetailsIcon`)?

Reply:
Yes, since these names do not escape this module, I did not care to invent a better name.  The linter gets angry with any names with prefix or suffix of underscore (`_`), so instead of adding a character, I removed one. I will add `Internal` as a suffix instead.

> However, since some of the styling is the same, couldn't you create a function that returns the div with the style and pass in the Icon element or prop (for `LinkIcon` and `DetailsIcon`)?

I think what you suggest will be just as much code as what I have done.

Reply:
I was thinking something like this :) But maybe not worth for two icons.
```
const IconWithStyle = (icon) => (
  <div
    style={{
      display: 'inline-block',
      margin: '-1rem 0',
      verticalAlign: 'middle',
    }}>
    {icon}
  </div>
);
```

Reply:
@sarah-clements 

I have a question about your suggestion:  Is styling an outer `div` that contains an inner component the same as styling the inner component? Since CSS is capable of insane inter dependencies; I am worried that more html tags, means more CSS styling opportunities, which results in more CSS rat-nest logic. This assumes I can even get the inner component to behave by styling the outer `div`.

On the other hand I have not seen this nightmare scenario in real life, so I wonder if there is some React magic happening here.

Reply:
I think this suggestion would be the same as embedding the component - you're styling the outer container and then any props passed directly to the component before its passed in would apply to that component's container (or whatever its used for).